tests/cpp/multidevice.h

```diff
@@ -88,17 +88,21 @@ class CommunicationTest
 class PipelineTest : public MultiDeviceTest {
  protected:
   void SetUp() override;
-  void validate();
+  // Run the multidevice fusion and stores the (sharded) results in "outputs"
   void execute();
-  void executeAndValidate() {
+  // Utility function used for validation in the tests. It compares the (sharded) outputs with ref_unsharded_outputs.
+  // if validate_with_prescribed_values= true, ref_unsharded_outputs is assumed to be set manually in the test body. Otherwise, ref_unsharded_outputs is computed by running a Fusion on a single device with the unsharded_inputs
```

Comment:
Thanks for the comments! 

```suggestion
  // if validate_with_prescribed_values is true, ref_unsharded_outputs is assumed to be set manually in the test body. Otherwise, ref_unsharded_outputs is computed by running a Fusion on a single device with the unsharded_inputs
```